src/otx/core/model/module/anomaly/anomaly_lightning.py

```diff
@@ -85,7 +85,7 @@ def __init__(
             metadata=metadata,
         )
 
-    def to_openvino(
+    def to_openvino(  # type: ignore[override]
```

Comment:
What is the problem with overriding?

Reply:
Due to `export_args`. I'll update it for anomaly default values.

Reply:
I updated it at https://github.com/openvinotoolkit/training_extensions/pull/2955/commits/979a01b5cfe421184ff9a21e4ab8e9ecfb84c185.